docker-compose.yml

```diff
@@ -7,24 +7,43 @@ services:
       - 8080:8080
     environment:
       - LOG_LEVEL=DEBUG
-      - MONGO_HOST=localhost
+      - MONGO_HOST=db
       - MONGO_PORT=27017
       - MONGO_USERNAME=admin
       - MONGO_PASSWORD=admin
-      - RABBIT_HOST=localhost
+      - RABBIT_HOST=rabbitmq
       - RABBIT_PORT=5672
       - RABBIT_USERNAME=guest
       - RABBIT_PASSWORD=guest
       - DATASET_HARVESTER_BASE_URI=https://datasets.staging.fellesdatakatalog.digdir.no
+    depends_on:
+      - db
+      - rabbitmq
+    networks:
+      fdk:
+        aliases:
+          - fdk-metadata-quality-service
   db:
     image: mongo
     ports:
       - 27017:27017
     environment:
       - MONGO_INITDB_ROOT_USERNAME=admin
       - MONGO_INITDB_ROOT_PASSWORD=admin
+    networks:
+      fdk:
+        aliases:
+          - db
   rabbitmq:
     image: rabbitmq:management
     ports:
       - 5672:5672
       - 15672:15672
+    networks:
+      fdk:
+        aliases:
+          - rabbitmq
+
+networks:
```

Comment:
Er dette nødvendig?

Reply:
Ikke nødvendig, men sørger for at containere joiner samme nettverk. Dette kan gjøre ting enklere hvis man f.eks. kjører docker-compose filer fra forskjellige repositorier, som i utgangspunktet lager hver sitt nettverk. Med denne er de garantert å kjøre i samme nettverk og man kan faktisk bruker interne hostnavn på tvers.